### PR TITLE
7 domains

### LINKED ISSUE #4226: brawlinthefamily.keenspot.com
### Basic Information

**Test link:** `http://brawlinthefamily.keenspot.com/comic/498-thwompstrategy/`
**Category:** ads
**Reported from:** Canada
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.149

### Additional Message

ads placeholder

### PR DESCRIPTION
**Pull request by DandelionSprout**
* Fix https://github.com/NanoMeow/QuickReports/issues/4226

Example pages:

```
! https://www.game-debate.com/blog/17014/undervolting-i7-4770k
game-debate.com##.bxz:has(:scope > div[id^="div-gpt-ad"])
! http://www.gamersheroes.com/game-guides/where-to-play-blackjack-in-red-dead-redemption-2/
||disqusservice.com/iframe/fallback^$subdocument,domain=gamersheroes.com
! http://keenspot.com/
keenspot.com###ad
! http://fallouttoyworks.keenspot.com/
fallouttoyworks.keenspot.com###atomekarect
fallouttoyworks.keenspot.com###atomekalead
fallouttoyworks.keenspot.com###atomekasky

! https://github.com/NanoMeow/QuickReports/issues/4226
! http://brawlinthefamily.keenspot.com/comic/498-thwompstrategy/
! http://countyoursheep.keenspot.com/
! http://nopinkponies.keenspot.com/
! http://thelounge.keenspot.com/
keenspot.com##div[id^="KS_G"]

! https://news.avclub.com/and-now-here-s-an-exhaustive-examination-of-every-job-h-1844245920
avclub.com##a[data-ga^="\[\[\"Commerce Inset"]:upward(2)
! https://bleedingcool.com/comics/marvel-comics-announces-upcoming-knull-crossover-event/
bleedingcool.com##.responsive-leaderboard
! https://www.freem.ne.jp/win/game/21911
freem.ne.jp##.ad-game-side
freem.ne.jp##.game-ad-bottom
```

### CODE REVIEW

File: dist/placeholder-buster.txt [hunk 1]
```diff
@@ -2017,6 +2017,23 @@ apachelounge.com##.type0:has(.adsbygoogle)
...
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/126
+keenspot.com###ad
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/126
+fallouttoyworks.keenspot.com###atomekarect
+fallouttoyworks.keenspot.com###atomekalead
+fallouttoyworks.keenspot.com###atomekasky
+
+# https://github.com/NanoMeow/QuickReports/issues/4226
+keenspot.com##div[id^="KS_G"]
```
Comment (jspenguin2017) [inline dist/placeholder-buster.txt]: Please fix these linting warnings: 
```
[WRN] 2030 : Rules for a domain is split into multiple sections
[WRN] 2035 : Rules for a domain is split into multiple sections
```

**Comment by DandelionSprout** [comment]
Oh, right. I had forgot about fixing this. Give me some minutes.

**Review by jspenguin2017 — CHANGES_REQUESTED** [review]
- `fallouttoyworks.keenspot.com` has another section on line `705`, please merge them
- Please provide test links for all rules

**Comment by DandelionSprout** [comment]
Will do. Give me 6min.

**Comment by DandelionSprout** [comment]
Should hopefully have been done now.

**Comment by jspenguin2017** [comment]
`https://www.game-debate.com/blog/17014/undervolting-i7-4770k`

**Comment by jspenguin2017** [comment]
`http://www.gamersheroes.com/game-guides/where-to-play-blackjack-in-red-dead-redemption-2/`

**Comment by jspenguin2017** [comment]
```
http://keenspot.com/
http://brawlinthefamily.keenspot.com/comic/498-thwompstrategy/
http://countyoursheep.keenspot.com/
http://nopinkponies.keenspot.com/
http://thelounge.keenspot.com/
```

**Comment by jspenguin2017** [comment]
`http://fallouttoyworks.keenspot.com/`

**Comment by jspenguin2017** [comment]
`https://news.avclub.com/and-now-here-s-an-exhaustive-examination-of-every-job-h-1844245920`

**Comment by jspenguin2017** [comment]
`https://bleedingcool.com/comics/marvel-comics-announces-upcoming-knull-crossover-event/`

**Comment by jspenguin2017** [comment]
`https://www.freem.ne.jp/win/game/21911`

**Pull request merged**